### DISCUSSION TITLE
Counting syllables in audio?

### DISCUSSION THREAD

**Discussion by jrappeneker** [openai/whisper#509] [Q&A] [answered]
Hi,

Would it be possible to use whisper to count the number of syllables uttered in an audio file?

**Comment by jongwook** [comment, marked as answer]
Whisper doesn't have the syllable count data in itself. I think the easiest way is to first run Whisper to get the transcript text and use a grapheme-to-phoneme (g2p) converter like [this](https://github.com/Kyubyong/g2p) to count vowels/diphthongs/etc in it.

**Reply by jrappeneker** [reply]
Thank you! I’ll try that :)

**Comment by EtienneAb3d** [comment]
See also text to IPA:
https://github.com/openai/whisper/discussions/318#discussioncomment-4103741
